velox/common/memory/Memory.h

```diff
@@ -85,15 +85,23 @@ class IMemoryManager {
   /// Returns the memory allocation alignment of this memory manager.
   virtual uint16_t alignment() const = 0;
 
-  /// Power users that want to explicitly modify the tree should get the root of
-  /// the tree.
+  /// Creates a memory pool for use with specified 'name', 'kind' and 'cap'. If
+  /// 'name' is missing, the memory manager generates a default name internally
```

Comment:
> generates a default name internally to ensure uniqueness

If 'name' is not empty, is the caller expected to ensure uniqueness?

Reply:
Changed to capacity?

Reply:
User might not care but we internally use atomic id to ensure the uniqueness for debugging purpose? Thanks!